package.json

```diff
@@ -14,6 +14,7 @@
     "is-reachable": "^5.0.0",
     "mocha": "^8.2.1",
     "pegjs": "^0.10.0",
+    "process-list": "^2.0.0",
```

Comment:
The other ps-list package is not needed anymore and can be removed

Reply:
I have removed the unused package